week-2-practise/level-3/5-indexOf.js

```diff
@@ -4,9 +4,21 @@
 // - insert `newValue` at that index
 // - NOT change the original array
 // - return a new array with the replacement value inserted
+function firstPartOfArr(arr, index) {
```

Comment:
Hey Kadir! What made you want to do this using lots of small functions instead of one big function? I really love this method. It makes the code really easy to read and understand.

Reply:
Making my code easy to read and understand was one of the reasons as that helps you a lot to understand easily when you revisit your code after awhile. The other reason was, I realised that I can use the same function in ```.slice()``` and ```.indexOf()``` exercise at level2 and level3. I planned to define and ```export``` these function inside the first exercise and use these functions inside the other exercises without defining by ```import```.  As I needed to more research for usage import/export statements, I could implement those changes today.